Pod/Classes/UI/CardTextField.swift

```diff
@@ -283,6 +309,18 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         monthTextField?.deleteBackwardCallback = {_ -> Void in self.numberInputTextField?.becomeFirstResponder()}
         yearTextField?.deleteBackwardCallback = {_ -> Void in self.monthTextField?.becomeFirstResponder()}
         
+        if isRightToLeftLanguage {
+            cvcTextField.textAlignment = .Left
+            monthTextField.textAlignment = .Left
+            yearTextField.textAlignment = .Right
+            numberInputTextField.textAlignment = .Right
+        } else {
+            cvcTextField.textAlignment = .Right
+            monthTextField.textAlignment = .Right
+            yearTextField.textAlignment = .Left
+            numberInputTextField.textAlignment = .Left
```

Comment:
seems that you changed the xib to use `textAlignment="natural"`, why do you need this snippet then? Is it to support version prior to iOS 9?